### PR TITLE
Feature/traveler iteration

### PR DESCRIPTION
**Pull request by Hpablo08**
## Category:
[Feature]

## Topic:
Created a trip request form
Created functions to receive from input
Created functions to display estimate of new trip

## What was changed:
* scipts.js
* html
* traveler

## Why:
-Allow user to select date, duration and number of travelers
-Allow user to see estimate of new trip
-Allow user to book trip and view on dashboard as pending

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -73,29 +79,23 @@ function displayTravelerData() {
...
+    const travelerDestinations = randomTraveler.destinations.find(destination => trip.destinationID === destination.id)
+    if (trip.status === 'pending') {
+      createTripCards('Pending Trip', travelerDestinations, trip)
+    } else if (trip.date < todaysDate) {
+      createTripCards('Past Trip', travelerDestinations, trip)
+    } else {
+      createTripCards('Upcoming Trip', travelerDestinations, trip)
+    }
+  })
+}
```
Comment (piperjarrett) [inline src/scripts.js:94]: I like how you have this!

File: src/scripts.js [hunk 2]
```diff
@@ -108,37 +108,76 @@ function displayTripCards(status, travelerDestinations, trip) {
...
+    id: Date.now(),
+    userID: randomTraveler.id,
+    destinationID: matchDestinationId.id,
+    travelers: parseInt(numOfTravelers.value),
+    date: tripDate.value.split('-').join('/'),
+    duration: parseInt(tripDuration.value),
+    status: 'pending',
+    suggestedActivities: []
+  }
+    console.log(travelerInputData)
```
Comment (piperjarrett) [inline src/scripts.js:137]: remember to delete this at the end

**Review by piperjarrett — COMMENTED** [review]
Looks good, ready to merge! just a few console.logs and comments you will want to delete when you're done!!

File: src/scripts.js (hunk 2 above)
Comment (Hpablo08) [reply src/scripts.js:137]: WILL DO!

**Pull request merged**